### PR TITLE
alter person mutations and create pet

### PR DESCRIPTION
**Pull request by Bastiani**

### CODE REVIEW

File: src/mutation/DeletePersonMutation.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { GraphQLString } from 'graphql';
+import { mutationWithClientMutationId } from 'graphql-relay';
+import { Person } from '../model';
+
+export default mutationWithClientMutationId({
+  name: 'DeletePerson',
+  inputFields: {
+    _id: {
+      type: GraphQLString,
```
Comment (sibelius) [inline src/mutation/DeletePersonMutation.js]: it should be required

File: src/mutation/DeletePersonMutation.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    },
+  },
+  mutateAndGetPayload: async ({ _id }) => {
+    const person = await Person.findOne({ _id });
+
+    if (person) {
+      return person
+        .remove()
+        .then(() => ({
```
Comment (sibelius) [inline src/mutation/DeletePersonMutation.js]: avoid then and catch, embrace async await

Comment (Bastiani) [reply src/mutation/DeletePersonMutation.js]: please, explain to me, "person" return a promise right? i not understand why use async/await in this case

Comment (jgcmarins) [reply src/mutation/DeletePersonMutation.js]: async/await is es6 way to handle promises

Comment (Bastiani) [reply src/mutation/DeletePersonMutation.js]: https://hackernoon.com/6-reasons-why-javascripts-async-await-blows-promises-away-tutorial-c7ec10518dd9

File: package.json [hunk 3]
```diff
@@ -9,6 +9,7 @@
     "dataloader": "^1.3.0",
     "dotenv-safe": "^4.0.4",
     "graphql": "^0.11.7",
+    "graphql-compose": "^2.10.1",
```
Comment (jgcmarins) [inline package.json]: I guess you don't need this

**Pull request merged**